roles/iridl/templates/ingrid-localdefs.tex

```diff
@@ -1,19 +1,11 @@
 \documentclass{article}
 
 \begin{document}
-redirectport is the optional specification of the port to redirect to.
-The default value is the port the server is running on.
-
-In this case, if the port is less than 80, I redirect to 80, otherwise
-I use it (so my experimental version can run on high ports and
-redirect to themselves, the production version redirects to the cache).
 \begin{ingrid}
-%/bindhost (localhost) def
 /redirectport {
-WWWinfo /port known
-{WWWinfo .port dup 85 lt { pop 80} if}
-{80} ifelse
-} def
+  WWWinfo .Host (:) search
+  { pop pop cvi } { pop 80 } ifelse
+}
```

Comment:
It sure is. And yet it seemed to work when I tested it. I think what happened is that this definition didn't take effect, so we fell back to the definition in [serverfns.tex](https://bitbucket.org/iridl/ingrid/src/ec180fd8c4e83736c103979f07010aa241138d73/src/serverfns.tex#lines-69), which is fine as long as there's no explicit port number in the URL. I've only deployed to Mali so far, and they use port 80. Tomorrow morning I'm deploying to Ethiopia, which is on port 8082, so it would have broken there. Thanks!